packages/@ember/deprecated-features/index.ts

```diff
@@ -10,3 +10,4 @@ export const ROUTER_EVENTS = !!'3.9.0';
 export const TRANSITION_STATE = !!'3.9.0';
 export const COMPONENT_MANAGER_STRING_LOOKUP = !!'4.0.0';
 export const JQUERY_INTEGRATION = !!'3.9.0';
+export const ALIAS_METHOD = !!'4.0.0';
```

Comment:
This is wrong (as is COMPONENT_MANAGER_STRING_LOOKUP above) the value should be the version that _introduced_ the deprecation (not the version that will remove it).

It should be (`!!'3.9.0'`)...

Reply:
Ahh, gotcha